src/nci/NfcTagManager.cpp

```diff
@@ -379,8 +401,15 @@ bool NfcTagManager::doTransceive(const std::vector<uint8_t>& command,
       uint8_t* cmd = new uint8_t[size];
       std::copy(command.begin(), command.end(), cmd);
 
-      tNFA_STATUS status = NFA_SendRawFrame(cmd, size,
-                             NFA_DM_DEFAULT_PRESENCE_CHECK_START_DELAY);
+      if (isMifareTech(tag.mTechLibNfcTypes[0])) {
+#ifdef NFCC_PN547
+        status = EXTNS_MfcTransceive(cmd, size);
```

Comment:
I think you removed the definition of 'status',
wouldn't this cause compilation error?